docs/how-to/setup-time-sources.md

```diff
@@ -0,0 +1,28 @@
+# How to: Setup Time Sources
```

Comment:
I would rephrase this title as `# How to set up time sources`

At the very least, I recommend removing the colon and splitting setup into two words

Reply:
Updated, thanks!